recipes/python-neovim/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "neovim" %}
+{% set version = "0.1.13" %}
+{% set sha256 = "fd41cef3caa34a8b685d2c0603022fb352e5c2f07c666594215d4a998629f35f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - msgpack-python >=0.4.0
+    - greenlet
+    - trollius  # [py27]
```

Comment:
Do we need all these at build time? Guessing we only need `python` and `setuptools`. Can we please update the listing in `build` accordingly?